mutt is calling mutt_perror twice, so the error messages are
sometimes confused, see upstream #3268

--- a/mx.c
+++ b/mx.c
@@ -415,7 +415,6 @@
   {
     dprint (1, (debugfile, "mx_get_magic(): unable to open file %s for reading.\n",
 		path));
-    mutt_perror (path);
     return (-1);
   }
 
